Src/ILGPU.Algorithms.Tests/TempViewManagerTests.cs

```diff
@@ -10,24 +10,25 @@ public abstract partial class TempViewManagerTests : TestBase
     {
         protected TempViewManagerTests(ITestOutputHelper output, TestContext testContext)
             : base(output, testContext)
-        { }
+        {
```

Comment:
ISSUE: Use `{ }` for empty implementation instead of `{` and `}` on separate lines.